python/cudf/cudf/core/abc.py

```diff
@@ -28,6 +28,11 @@ def deserialize(cls, header, frames):
         pass
 
     def device_serialize(self):
+        """Serialize into pickle format of device object
```

Comment:
Modified the serialization docs

Reply:
Perhaps we shouldn't document the `*serialize` methods of each of the subclasses of `Serializable`? These are private methods and their docstrings look essentially identical. If we ever change the way `Serializable` works, we'll have to duplicate the doc change in about a dozen different places.

Reply:
In that case, we could use `copy_docstring` actually

Reply:
`copy_docstring` is an option, but the docstring isn't technically needed as we don't publish these methods, correct?

We should document the methods of the base class `Serializable`. They will still show up as the docstring for subclasses in IPython/Jupyter/wherever so that developers -- us -- can look them up. Does that sound OK?

Reply:
Sure, removed the docstrings from inherited classes.